### DISCUSSION TITLE
Can some further details be provided on the ClassPath requirements

### DISCUSSION THREAD

**Discussion by DamianReeves** [scalacenter/tasty-query#349] [Q&A]
I'm trying to evaluate tasty-query but I'm still unclear on some things.

Perhaps some examples of parameters to the example that will work would be helpful?

The ReadMe says something about using `jrt:`, can any clarity be provided on that? 
Every time I use this do I also need to provide the classpath for the scala library as well? If so how do I do that?
This is all really hazy to me.

**Comment by sjrd** [comment]
Indeed, you need to provide every classpath entry that is required for the program to be valid. That includes the JDK (with `jrt:`), and the Scala standard library.

The Scala standard library is no different than any other `.jar` file on the classpath. You find its fully-qualified path on your system, and you wrap it in a `Path`, like any other .jar file. Usually, your build tool will tell you exactly where it is. For the JDK, use the exact incantation provided in the readme.

**Reply by anjunar** [reply]
@DamianReeves I got another approach to load all classes, because I am using the Wildfly Server.

https://github.com/anjunar/scala-reflections

...hope this helps...

**Reply by DamianReeves** [reply]
Thanks this looks promising.

**Reply by anjunar** [reply]
I am at the beginning... have to write tests now... but I think this kind of classloading is much easier

**Comment by sjrd** [comment]
@DamianReeves Have your questions been answered here? Can I close this discussion?

**Reply by DamianReeves** [reply]
Yes this may be closed, thanks
